doc/modules/ROOT/pages/getting-started.adoc

```diff
@@ -38,6 +38,9 @@ assert gds.version()
 "2.1.9"
 ----
 
+Please note that the `GraphDataScience` object needs to communicate with a Neo4j database upon construction, and uses the default "neo4j" database by default.
+If there is no such database, you will need to <<specifying-targeted-database, provide a valid database using the `database` keyword>>.
```

Comment:
```suggestion
If there is no such database, you will need to <<specifying-targeted-database, provide a valid database using the `database` parameter>>.
```

Reply:
Made it "keyword parameter" now to be super clear